x-pack/plugins/secops/public/components/visualization_placeholder.tsx

```diff
@@ -26,27 +29,52 @@ export const VisualizationPlaceholder = styled(EuiPanel)`
   }
 `;
 
+export const ProviderContainer = styled.div`
+  margin: 5px;
+  user-select: none;
+  cursor: grab;
```

Comment:
I take it you want this `cursor: grab;` with this PR at this moment and this isn't an accidental left over from when you were playing with things. Just pointing it out in case you didn't want it here.

Reply:
Yes, the intent is to have it there until part-2 is merged. 🦅 👀